cloudtools/aws/spot.py

```diff
@@ -300,21 +315,23 @@ def __cmp__(self, other):
 
 
 def get_spot_choices(connections, rules, product_description, start_time=None,
-                     instance_type=None):
+                     moz_instance_type=None):
```

Comment:
I don't see `moz_instance_type` used anywhere. Can you drop it?

Also, while you are here, can you also drop the useless __main__block at the bottom?